Automattic/gravatar-enhanced
# Woo: Account details avatar init

In this PR, we add the initial version of the Gravatar and Gravatar Quick Editor integration in the Woo Account details page:

![image](https://github.com/user-attachments/assets/f9ed16bd-f043-4f8f-9bad-c7e1fdeac88b)

It adds your avatar above the Woo account navigation menu and after hovering over the avatar and clicking on it, it opens the quick editor where you can change the avatar.

Props @jcheringer for most of the work/code.

## Testing instructions

Using a local WP site, install Woo. Navigate to `/my-account/edit-account/` and make sure you can see either the default avatar or your based on the user email you have in your local WP setup. If you have some kind of non-existent email or a local one, it will show the default avatar.
